src/components/navigation/footer/Footer.tsx

```diff
@@ -60,10 +60,19 @@ const Footer: Footer = ({ className, children, visuallyHiddenText = 'Support lin
   const footerCols = Children.toArray(children).filter((child) =>
     childIsOfComponentType(child, FooterList),
   );
+  const footerCopyright = Children.toArray(children).filter((child) =>
+    childIsOfComponentType(child, FooterCopyright),
+  );
 
   let newChildren = children;
```

Comment:
it's been raised on sonar - but this assignment is pointless now given the if/else addition